portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,102 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Hello, my name is Sydney Beasley</title>
+    <title>Sydney Beasley Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color:rgb(233, 206, 255);">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>
+        <intro-title> Hello, my name is Sydney!</intro-title>
```

Comment:
Sorry if this comment comes through in two different forms, but it looks like I lost my original...

I'd actually suggest that you don't need the custom tags you've made, particularly `intro-title` and `cursive`. Some reasons: 1) in your case, it's simple and direct to just style `h1` and `h2` in your CSS. `h1` already suggests that it's a top-level heading like a title, so having the extra tag to describe it may not be too useful. 2) Regarding the `cursive` tag, usually it's a good idea not to have anything in your HTML that suggests visual appearance, that way you can choose to change that appearance in the future (let's say by un-cursiving the text) by just updating one CSS rule, and you don't have to go through and change every instance you've added `cursive` in the HTML.